src/main/java/Exercise12.java

```diff
@@ -0,0 +1,17 @@
+public class Exercise12 {
+    public int calculateFactorial(int number) {
+        int result = 1;
+        for (int i = 2; i <= number; i++) {
+            result *= i;
+        }
+        return result;
+    }
+
+    public double calculateSum(int number) {
+        double result = 1;
+        for (int i = 1; i <= number; i++) {
+            result += (double) 1 / calculateFactorial(i - 1) / i;
```

Comment:
Có cách nào bỏ hẳn hàm calculateFactorial đi được không?

Reply:
Em có sửa lại rồi ạ

Reply:
That's sửa rồi 👎

Reply:
Dạ em sửa rồi ạ.